src/components/SystemResponse/hooks.ts

```diff
@@ -42,7 +42,7 @@ export const useAnimatedMessages = ({
   onAnimationEnd: VoidFunction;
 }) => {
   const shouldAnimate = isLive && messages.length;
-  const [showIndicator, setShowIndicator] = useState(false);
+  const [showIndicator, setShowIndicator] = useState(true);
```

Comment:
this has to be true otherwise there's one render cycle in between `useRuntime` and the `useEffect` where the indicator doesn't load.
This caused a flicking effect.